astropy/wcs/__init__.py

```diff
@@ -1,21 +1,24 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 """
 .. _wcslib: http://www.atnf.csiro.au/~mcalabre/WCS/
-.. _Paper IV: http://www.atnf.csiro.au/people/mcalabre/WCS/index.html
+.. _WCS papers: http://www.atnf.csiro.au/people/mcalabre/WCS/index.html
+.. _Paper IV: http://www.atnf.csiro.au/people/mcalabre/WCS/dcs_20040422.pdf
```

Comment:
The time coordinates paper is called now `Paper IV`. As far as I can tell the distortion paper is not numbered, so may be just call it the "Distortion paper". I can't think of a good short name. There are also a few other places which reference it as Paper IV.